pkg/infra/pulumi_aws/iac/load_balancing.ts

```diff
@@ -27,14 +27,15 @@ export class LoadBalancerPlugin {
     }
 
     public createLoadBalancer = (
+        appName: string,
         resourceId: string,
         params: LoadBalancerArgs
     ): aws.lb.LoadBalancer => {
         let lb: aws.lb.LoadBalancer
         switch (params.loadBalancerType) {
             case 'application':
-                lb = new aws.lb.LoadBalancer(`${resourceId}-alb`, {
-                    name: `${resourceId}-alb`,
+                lb = new aws.lb.LoadBalancer(`${appName}-${resourceId}-alb`, {
+                    name: `${appName}-${resourceId}`,
```

Comment:
Will this (and similar) conflict with @DavidSeptimus-Klotho's work? If so, probably just need to coordinate.

Reply:
i mentioned i was doing this to him, he said he hadnt touched these modules yet so should be good